src/osdag/design_type/compression_member/Column.py

```diff
@@ -1134,138 +1155,166 @@ def results(self):
 
-                self.result_nd_esr_zz = self.optimum_section_cost_results[self.result_cost]['ND_ESR_zz']
-                self.result_nd_esr_yy = self.optimum_section_cost_results[self.result_cost]['ND_ESR_yy']
+        #        self.result_ebs_zz = self.optimum_section_cost_results[self.result_cost]['EBS_zz']
+        #        self.result_ebs_yy = self.optimum_section_cost_results[self.result_cost]['EBS_yy']
 
-                self.result_phi_zz = self.optimum_section_cost_results[self.result_cost]['phi_zz']
-                self.result_phi_yy = self.optimum_section_cost_results[self.result_cost]['phi_yy']
+        #        self.result_nd_esr_zz = self.optimum_section_cost_results[self.result_cost]['ND_ESR_zz']
+        #        self.result_nd_esr_yy = self.optimum_section_cost_results[self.result_cost]['ND_ESR_yy']
 
-                self.result_srf_zz = self.optimum_section_cost_results[self.result_cost]['SRF_zz']
-                self.result_srf_yy = self.optimum_section_cost_results[self.result_cost]['SRF_yy']
+        #        self.result_phi_zz = self.optimum_section_cost_results[self.result_cost]['phi_zz']
+        #        self.result_phi_yy = self.optimum_section_cost_results[self.result_cost]['phi_yy']
 
-                self.result_fcd_1_zz = self.optimum_section_cost_results[self.result_cost]['FCD_1_zz']
-                self.result_fcd_1_yy = self.optimum_section_cost_results[self.result_cost]['FCD_1_yy']
+        #        self.result_srf_zz = self.optimum_section_cost_results[self.result_cost]['SRF_zz']
+        #        self.result_srf_yy = self.optimum_section_cost_results[self.result_cost]['SRF_yy']
 
-                self.result_fcd_2 = self.optimum_section_cost_results[self.result_cost]['FCD_2']
+        #        self.result_fcd_1_zz = self.optimum_section_cost_results[self.result_cost]['FCD_1_zz']
+        #        self.result_fcd_1_yy = self.optimum_section_cost_results[self.result_cost]['FCD_1_yy']
 
-                self.result_fcd_zz = self.optimum_section_cost_results[self.result_cost]['FCD_zz']
-                self.result_fcd_yy = self.optimum_section_cost_results[self.result_cost]['FCD_yy']
+        #        self.result_fcd_2 = self.optimum_section_cost_results[self.result_cost]['FCD_2']
 
-                self.result_fcd = self.optimum_section_cost_results[self.result_cost]['FCD']
-                self.result_capacity = self.optimum_section_cost_results[self.result_cost]['Capacity']
+        #        self.result_fcd_zz = self.optimum_section_cost_results[self.result_cost]['FCD_zz']
+        #        self.result_fcd_yy = self.optimum_section_cost_results[self.result_cost]['FCD_yy']
 
-                print(f"design_status_list2{self.design_status}")
+        #        self.result_fcd = self.optimum_section_cost_results[self.result_cost]['FCD']
+        #        self.result_capacity = self.optimum_section_cost_results[self.result_cost]['Capacity']
 
+        #        print(f"design_status_list2{self.design_status}")
```

Comment:
That's a big chunk of commented and changed code. Please add a comment as to why it still remains and what to do with it.

Reply:
This code portion was commented, in the flexure files. I had to make a similar common_result function in the Column file. This portion needs to remain, I felt, as it will help when later other section profiles are integrated, for debugging/using the resulting values.